Tests/js/views/tabs/assets/ez-locationviewlocationstabview-tests.js

```diff
@@ -845,6 +846,65 @@ YUI.add('ez-locationviewlocationstabview-tests', function (Y) {
         }
     });
 
+    swapLocationTest = new Y.Test.Case({
+        name: "eZ LocationViewLocationsTabView swap location test",
+        setUp: function () {
+            this.contentMock = new Mock();
+            this.locations = [this.locationMock, this.locationMock];
+            this.resources = {
+                MainLocation: '/main/location/id'
+            };
+
+            Mock.expect(this.contentMock, {
+                'method': 'get',
+                'args': ['resources'],
+                returns: this.resources
+            });
+
+            this.view = new Y.eZ.LocationViewLocationsTabView({
+                content: this.contentMock,
+                container: '.container'
+            });
+        },
+
+        tearDown: function () {
+            this.view.destroy();
+            delete this.view;
+        },
+
+        "Should fire `swapLocation` event": function () {
+            var eventFired = false,
+                that = this,
+                locationIdForSwap,
+                swapButton;
+
+            this.view.render();
+
+            swapButton = this.view.get('container').one('.ez-locations-swap-button');
+            locationIdForSwap = swapButton.getAttribute('data-location-id');
```

Comment:
not used anywhere, I guess it's a left over of a previous of the patch. You should remove that variable and the corresponding data attribute from the template.